src/sentry/static/sentry/app/views/performance/transactionSummary/index.tsx

```diff
@@ -75,6 +76,13 @@ class TransactionSummary extends React.Component<Props, State> {
         this.fetchTotalCount();
       }
     }
+
+    if (
+      !isEqual(prevProps.selection.projects, selection.projects) ||
+      !isEqual(prevProps.selection.datetime, selection.datetime)
+    ) {
+      loadOrganizationTags(api, organization.slug, selection);
```

Comment:
This was an oversight in the transaction summary changes. Changing the datetime/project selector should update the tag list.